### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by gastaldi**
It would be nice to have a code of conduct for contributors - https://opensource.guide/code-of-conduct/

### CODE REVIEW

**Comment by FroMage** [comment]
I personally find the contributor covenant to be a very bad code of conduct. If we have to pick one, can't we pick another one?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+## Scope
+
+This Code of Conduct applies both within project spaces and in public spaces when an individual is representing the project or its community. Examples of representing a project or community include using an official project e-mail address, posting via an official social media account, or acting as an appointed representative at an online or offline event. Representation of a project may be further defined and clarified by project maintainers.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be reported by contacting the project team at https://quarkusio.zulipchat.com/. The project team will review and investigate all complaints, and will respond in a way that it deems appropriate to the circumstances. The project team is obligated to maintain confidentiality with regard to the reporter of an incident. Further details of specific enforcement policies may be posted separately.
```
Comment (gsmet) [inline CODE_OF_CONDUCT.md]: I don't have any opinion about this but if we include something like this, the point of contact should be a private email address. Not Zulip as you don't know who you should contact on Zulip and you probably don't want to make your complaint public.

Comment (gastaldi) [reply CODE_OF_CONDUCT.md]: Right,. which email should be used here?

**Comment by gastaldi** [comment]
@FroMage do you have an example of a good code of conduct doc?

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (gsmet) [reply CODE_OF_CONDUCT.md]: I think we will need to create a specific one. But better ask @emmanuelbernard about this before spending too much time on it.

**Comment by emmanuelbernard** [comment]
Hello,

This is a topic we have started discussions on but we want to take our time to mature it.
The Quarkus community at large has been extremely welcoming and friendly, so we are on the right track.

In the mean time I will close this PR while this is decanted.

**Pull request closed without merging**